recipes/phyloacc/meta.yaml

```diff
@@ -0,0 +1,66 @@
+#######################################################################
+# Conda recipe for installing PhyloAcc
+# Supported platforms: Linux
+# Author: Gregg Thomas (gwct@github)
+#######################################################################
+
+{% set name = "PhyloAcc" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "1500ae0a562ca1c8d55bcd081b6d2da27d77657b6cc5811fe8b540b61bb31621" %}
+# Setting some Jinja variables for the recipe
+# sha256 from openssl sha256 <release tarball>
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+# Package info
+
+source:
+  url: https://github.com/phyloacc/PhyloAcc/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+# Source location
+
+build:
+  number: 0
+# Build info
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - gsl
+    - openblas
+    - liblapack
+    - armadillo
+    - python >=3.6
```

Comment:
Please remove ">=3.6" here and from the "run:" section. Then under "build:" on line 23 put "skip: True  # [py2k]`.

Reply:
Changes made in [ecf5e4a](https://github.com/bioconda/bioconda-recipes/pull/33464/commits/ecf5e4a4d8ecbf7001dae677576f34f73ae0222d). Thanks!